pom.xml

```diff
@@ -239,6 +239,11 @@
       <groupId>org.xerial.snappy</groupId>
       <version>1.1.10.6</version>
     </dependency>
+    <dependency>
+      <groupId>com.github.luben</groupId>
+      <artifactId>zstd-jni</artifactId>
+      <version>1.5.7-4</version>
```

Comment:
Freeze `zstd-jni`'s version so it's matching with the native libraries stored under `src/main/resources`.